README.md

```diff
@@ -1,5 +1,4 @@
-# DSP-UI library (knora-ui)
-
+# DSP-UI library (@knora/ui)
```

Comment:
Shouldn't this go into `projects/knora-ui/README.md``

Reply:
Probably yes

Reply:
https://github.com/dasch-swiss/knora-ui-ng-lib/commit/4ee6d7a8836b5be8dc88fa7b356a6528897fbf3e